scipy/integrate/_bvp.py

```diff
@@ -1097,34 +1108,34 @@ def solve_bvp(fun, bc, x, y, p=None, S=None, fun_jac=None, bc_jac=None,
             status = 1
             if verbose == 2:
                 nodes_added = "({})".format(nodes_added)
-                print_iteration_progress(iteration, max_rms_res, m,
+                print_iteration_progress(iteration, max_res, m,
                                          nodes_added)
             break
 
         if verbose == 2:
-            print_iteration_progress(iteration, max_rms_res, m, nodes_added)
+            print_iteration_progress(iteration, max_res, m, nodes_added)
 
         if nodes_added > 0:
             x = modify_mesh(x, insert_1, insert_2)
             h = np.diff(x)
             y = sol(x)
-        else:
+        elif max_res <= tol:
```

Comment:
Here we just check `max_bc_res < bc_tol`.

Reply:
That change has been made.